src/FilterChain.php

```diff
@@ -70,7 +70,8 @@ public function __construct($options = null)
     }
 
     /**
-     * @deprecated This method will be removed in 3.0.0 without replacement
+     * @deprecated This method will be removed in 3.0.0 without replacement. Future versions of FilterChain will require
```

Comment:
An alternative can be a link to the migration guide.

Reply:
Links in addition to the message would be good when we have a link.

Reply:
The "preparing-for" docs are out of date right now, and the migration guide un-released so I've merged as-is for now